src/microbe_stage/Microbe.cs

```diff
@@ -1089,13 +1089,15 @@ public void ApplyPropertiesFromSave(Microbe microbe)
     internal void SuccessfulScavenge()
     {
         GameWorld.AlterSpeciesPopulation(Species,
-            Constants.CREATURE_SCAVENGE_POPULATION_GAIN, "successful scavenge");
+            Constants.CREATURE_SCAVENGE_POPULATION_GAIN,
+            TranslationServer.Translate("SUCCESSFUL_SCAVENGE"));
```

Comment:
I still think the translation should happen when printing this information, so this could be:
```
            Constants.CREATURE_SCAVENGE_POPULATION_GAIN,
            "SUCCESSFUL_SCAVENGE");
```

Reply:
I tried it, the issue is that then, the translation system cannot find the string and remove it from the .pot file.

Reply:
Ah, that's true. I didn't consider that. Then it'll need to be this way. Let's hope the translation server is fast enough to not cause laggy frames when translating stuff on the fly.